ios/OX Coi/Application/AppDelegate.swift

```diff
@@ -34,8 +36,35 @@ class AppDelegate: FlutterAppDelegate {
                 } else {
                     result(nil)
                 }
+            } else if call.method.contains(Method.Sharing.SendSharedData) {
+                self.shareFile(arguments: call.arguments as! Dictionary<String, String>)
```

Comment:
If you declare `call.arguments` as `[String: String]`, then you can...

Reply:
Done

Reply:
Just one thing I just was thinking about...  
Let's assume `call.arguments` isn't a dictionary or it is even `nil`. What will happen if we do a force cast here? Right, the app will crash. That's not what we like to expect. So I suggest to use a `guard let ...` instead to be on the safe side.